packages/datasets-fs/src/commands/generateV1.ts

```diff
@@ -0,0 +1,207 @@
+import { CaptchaTypes, CaptchaWithoutId, Captchas, CaptchasContainerSchema, Item, RawSolution } from '@prosopo/types'
+import { Generate, ArgsSchema as GenerateArgsSchema } from './generate.js'
+import { ProsopoEnvError } from '@prosopo/common'
+import { at, get, lodash } from '@prosopo/util'
+import { blake2AsHex } from '@polkadot/util-crypto'
+import { z } from 'zod'
+import bcrypt from 'bcrypt'
+import cliProgress from 'cli-progress'
+import fs from 'fs'
+
+export const ArgsSchema = GenerateArgsSchema.extend({
+    solved: z.number().optional(),
+    unsolved: z.number().optional(),
+    minCorrect: z.number().optional(),
+    maxCorrect: z.number().optional(),
+})
+export type ArgsSchemaType = typeof ArgsSchema
+export type Args = z.infer<ArgsSchemaType>
+
+export class GenerateV1 extends Generate<ArgsSchemaType> {
+    public override getArgSchema() {
+        return ArgsSchema
+    }
+
+    public override getDescription(): string {
+        return 'Generate distinct captchas producing captcha challenges comprising 2 rounds, one labelled and one unlabelled'
+    }
+
+    public override getOptions() {
+        return lodash().merge(super.getOptions(), {
+            solved: {
+                description: 'Number of captchas to generate that are solved',
+                number: true,
+            },
+            unsolved: {
+                description: 'Number of captchas to generate that are unsolved',
+                number: true,
+            },
+            minCorrect: {
+                description: 'Minimum number of target images in each captcha',
+                number: true,
+            },
+            maxCorrect: {
+                description: 'Maximum number of target images in each captcha',
+                number: true,
+            },
+        })
+    }
+
+    public override async _run(args: Args) {
```

Comment:
helper functions would be useful for readability here - _run is about 150 lines long.

Might not be worth doing if we aren't looking to be using v1 captcha generation in the future though